docs/flask_config.md

```diff
@@ -81,53 +80,54 @@ report to provide context.
 - You have extended table information that is applicable to your datastore which you want to scrape and provide in the
 table page
 
-Programmatic Descriptions are referred to by a "description source" which is a unique identifier.
-You can then configure the descriptions to have a custom order in the config.py file like so:
+Programmatic descriptions are referred to by a "description source" which is a unique identifier.
+In the UI, they will appear on the table page under structured metadata.
+
+In config.py you can then configure the descriptions to have a custom order, as well as whether or not they should exist in the left column or right column.
 ```    
 PROGRAMMATIC_DISPLAY = {
-           "s3_crawler": {
-               "display_order": 0
-           },
-           "quality_service": {
-               "display_order": 1
-           },
-           "doesnt_exist": {
-               "display_order": 2
-           }
-       }
+    'RIGHT': {
+      "test3" : {},
+      "test2" : { "display_order": 0 }
```

Comment:
so test2 should match the source field in programmatic description in backend?

Reply:
Yes, the original implementation used the `source` as a unique id. We could rethink this but the idea was that since engineers control what to ingest as the value of `source` to begin with, they can ensure they are unique.